common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -1527,7 +1527,9 @@ public enum PUBLIC_SHARE_VISIBILITY { samePrimaryDomain, all, none };
     public static final KnownKey zimbra_strict_unclosed_comment_tag = KnownKey.newKey(true);
     public static final KnownKey zimbra_skip_tags_with_unclosed_cdata = KnownKey.newKey("style");
 
-    
+    //ZCS-11191 : Allow to enable/disable SRS feature
+    public static final KnownKey zimbra_srs_enabled = KnownKey.newKey(false);
```

Comment:
any reason to put this into LC instead of LDAP? are we planning to release this feature in a patch?